src/io/flutter/run/daemon/FlutterDaemonService.java

```diff
@@ -68,14 +69,16 @@ public void processTerminated(ProcessHandler handler, FlutterDaemonController co
     void deviceRemoved(ConnectedDevice device);
   }
 
-  @Nullable
+  @NotNull
   public static FlutterDaemonService getInstance(@NotNull Project project) {
```

Comment:
Ah!  I see why you weren't checking this before.

It looks like you're you treating the case where `com.intellij.openapi.components.ServiceManager#doGetService` returns `null` as a kind of assert which works for me!